Apromore-Custom-Plugins/Process-Discoverer-Portal-Plugin/src/main/java/org/apromore/plugin/portal/processdiscoverer/components/ViewSettingsController.java

```diff
@@ -159,43 +177,44 @@ public void initializeControls(Object data) {
 
         frequencyAggSelector = (Combobox) compViewSettings.getFellow("frequencyAggSelector");
         durationAggSelector = (Combobox) compViewSettings.getFellow("durationAggSelector");
+        
+        primaryTypeLabel = FREQ_LABEL;
+        primaryAggregateCode = measures[0];
     }
-
+    
     @Override
     public void initializeEventListeners(Object data) { 
         perspectiveSelector.addEventListener("onSelect", new EventListener<Event>() {
```

Comment:
The half dozen or so event handler registrations appearing over the next 100 or so lines could be abbreviated with lambda expressions.

Reply:
The utmost importance in coding is the readability of the code which supports maintainabilty, instead of always using lambda. Lambda can make the code look nice in some cases but not always. Can you assess on the readability of the code here?

Reply:
The argument is that the type signature of addEventList already tells you that its last argument is going to be an EventListener, so explicitly writing out the inner class constructor (new EventListener) is uninformative.